test/driver.ts

```diff
@@ -119,7 +136,7 @@ export class Driver {
     }
 
     get rewardsGovernor(): Participant {
```

Comment:
Agree

Reply:
Actually, there was a reason - we used the ERC20 helpers in the Participant class.

Reply:
this method is unclear. the term governor was used instead of owner in some contracts. what does it mean here? 
but clearly, the `Participant` class is abused here - not any token holder is a validator with registration parameters.. is there a simple way to separate the usages: orbs holder, validator - which may be a subclass of holder,  contract owner (governor?)

Reply:
It's the address that is allowed to make reward-related governance decisions (such as setting rates). It is not necessarily the owner of the contract.
Regarding refactoring the Participant class, see my other comment on this.